src/main/code.ts

```diff
@@ -20,23 +20,26 @@ if (figma.currentPage.selection.length != 1) {
 	if (msg.type === 'build-table') {
 	  
 	  var textNode = getFirstChildOfTypeText(figma.currentPage.selection[0])
-	  if (textNode === undefined) {
+	  if (textNode === undefined || textNode == null) {
 		figma.closePlugin("🛑 Selected node has to child of type TEXT");
```

Comment:
Ah, here is the typo I was referring to, the other day. Can you please fix this as well?